### PR TITLE
Add instant respawn option

### PR DESCRIPTION
**Pull request by MisterRafers**
I think, name describes the option

### CODE REVIEW

**Comment by HafimieForever** [comment]
great idea

**Review by HiroMizuki — APPROVED** [review]

**Review by MasterDirector99 — APPROVED** [review]
Nice work!

**Comment by tposejank** [comment]
no

**Comment by fireredfan7** [comment]
> no

Why not?

**Comment by tposejank** [comment]
> > no
> 
> Why not?

you're too lazy to press enter as soon as you die???

**Review by Starmapo — APPROVED** [review]

**Review by crowplexus — APPROVED** [review]

**Review by tposejank — APPROVED** [review]

**Comment by Steviegt6** [comment]
> > > no
> > 
> > 
> > Why not?
> 
> you're too lazy to press enter as soon as you die???

It's less a matter of laziness and more a matter of patience. Respawning doesn't occur instantly by pressing "ENTER".

File: source/options/GameplaySettingsSubState.hx [hunk 1]
```diff
@@ -138,4 +145,4 @@ class GameplaySettingsSubState extends BaseOptionsMenu
 
 		super();
 	}
-}
\ No newline at end of file
+}
```
Comment (Steviegt6) [inline source/options/GameplaySettingsSubState.hx:148]: Accidental line ending change?

File: source/options/GameplaySettingsSubState.hx [hunk 2]
```diff
@@ -62,6 +62,13 @@ class GameplaySettingsSubState extends BaseOptionsMenu
 			'bool',
 			true);
 		addOption(option);
+		
+		var option:Option = new Option('Instant Respawn',
+			"If checked, you will be respawned automatically, without animation.",
```
Comment (Steviegt6) [inline source/options/GameplaySettingsSubState.hx:67]: Potentially change to: "If checked, you will automatically respawn, skipping the game over animation"?

**Review by steviegt6 — APPROVED** [review]
Looks good, just a few negligible things.

File: source/options/GameplaySettingsSubState.hx (hunk 1 above)
Comment (CerBor) [reply source/options/GameplaySettingsSubState.hx:148]: Oh, yea, i will fix it

File: source/options/GameplaySettingsSubState.hx (hunk 2 above)
Comment (CerBor) [reply source/options/GameplaySettingsSubState.hx:67]: OK, that will give more information, what this option does.

**Comment by MayoALT** [comment]
> no

someone could make a pr that cures cancer and ends world hunger and you would still say no to it

**Comment by HiroMizuki** [comment]
> > no
> 
> someone could make a pr that cures cancer and ends world hunger and you would still say no to it

i laughed at it so hard sorry

**Review by Lila-Rogers — APPROVED** [review]

**Pull request closed without merging**